erroranalysis/erroranalysis/_internal/cohort_filter.py

```diff
@@ -178,17 +178,20 @@ def build_query(filters, categorical_features, categories):
             arg0 = str(filter[ARG][0])
             colname = filter[COLUMN]
             if method == CohortFilterMethods.METHOD_GREATER:
-                queries.append("`" + colname + "` > " + arg0)
+                queries.append("`" + colname + "` greater than " + arg0)
```

Comment:
hmm if we are using string in backend now instead of operators then it seems like we need to localize them now.  I haven't seen a case like this before, where the string is constructed in backend but displayed in frontend.  I wonder if it might be easier to send the two columns and operator to frontend and let it localize them.